### PR TITLE
pfSense-pkg-RRD_Summary - Rewrite to include display of historical data

### PR DESCRIPTION
**Pull request by shonjir**
Rewrite of RRD Summary package to allow display of available historical data beyond current and previous month.  Bumped the major version because of the significant changes.

This is my first attempt at a pfSense package, please be gentle :)

### CODE REVIEW

**Comment by shonjir** [comment]
Screenshot of new display:
rrd screenshot

File: sysutils/pfSense-pkg-RRD_Summary/files/usr/local/www/status_rrd_summary.php [hunk 1]
```diff
@@ -20,40 +20,87 @@
...
+			if ($start < $first) $start = $first;
+			if ($end > $last) $end = $last;
+			$data = fetch_rrd_summary($rrd, "epoch+{$start}s", "epoch+{$end}s", $units, 24*60*60);
+			?>
+									<tr>
+										<td><?=date("Y-m-d", $start); ?> to <?=date("m-d-Y", $end); ?></td>
+										<td><?=sprintf("%0.2f %s", $data["total_in"], $u[$units]); ?></td>
+										<td><?=sprintf("%0.2f %s", $data["total_out"], $u[$units]); ?></td>
+										<td><?=sprintf("%0.2f %s", $data["total_in"] + $data["total_out"], $u[$units]); ?></td>
+									</tr>
```
Comment (sbeaver-netgate) [inline sysutils/pfSense-pkg-RRD_Summary/files/usr/local/www/status_rrd_summary.php]: Indenting looks excessive

Comment (shonjir) [reply sysutils/pfSense-pkg-RRD_Summary/files/usr/local/www/status_rrd_summary.php]: I agree, but felt it was better to preserve the original indenting than fix it here.  Indenting is fixable, let me know if you'd like me to fix it.

Comment (sbeaver-netgate) [reply sysutils/pfSense-pkg-RRD_Summary/files/usr/local/www/status_rrd_summary.php]: Yes please. It is good to take the opportunity to fix things like this and get the codebase as consistent as possible :) Nice changes BTW.

Comment (shonjir) [reply sysutils/pfSense-pkg-RRD_Summary/files/usr/local/www/status_rrd_summary.php]: Thanks!  Will make the changes.  Would you like me to squash the commit?

**Comment by shonjir** [comment]
@sbeaver-netgate - indentation ought to be ok now.

File: sysutils/pfSense-pkg-RRD_Summary/Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-RRD_Summary
-PORTVERSION=	1.3.2
-PORTREVISION=	2
+PORTVERSION=	2.0
+PORTREVISION=	1
```
Comment (rbgarga) [inline sysutils/pfSense-pkg-RRD_Summary/Makefile]: When PORTVERSION changes, PORTREVISION line must be removed

Comment (shonjir) [reply sysutils/pfSense-pkg-RRD_Summary/Makefile]: Done.

**Comment by shonjir** [comment]
@rbgarga - PORTREVISION removed as requested.

**Comment by jim-p** [comment]
In an effort to improve the tracking of changes and bug/feature requests, we have decided to require an entry on the [pfSense Redmine issue tracker](https://redmine.pfsense.org) associated with every pull request, and likewise the Redmine entry should also have a link back to the pull request.

If you could, please associate this PR to a Redmine issue either by locating an existing issue at https://redmine.pfsense.org or by creating a new issue. Add a link to the Redmine issue that points to this PR, and also add a link on the PR to the Redmine issue.

For more information, see https://doc.pfsense.org/index.php/Submitting_a_Pull_Request_via_Github

Thanks!

**Comment by shonjir** [comment]
Created feature issue https://redmine.pfsense.org/issues/8190.

File: sysutils/pfSense-pkg-RRD_Summary/Makefile [hunk 3]
```diff
@@ -1,8 +1,7 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-RRD_Summary
-PORTVERSION=	1.3.2
-PORTREVISION=	2
+PORTVERSION=	2.0
 CATEGORIES=	sysutils
```
Comment (shonjir) [inline sysutils/pfSense-pkg-RRD_Summary/Makefile]: Removed PORTREVISION

**Comment by shonjir** [comment]
Not sure what changes are still outstanding - I made the requested changes.  Is there anything else?

**Pull request merged**